script/ArcPay.s.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Script.sol";
+import "../src/ArcPay.sol";
+
+contract ArcPayScript is Script {
+    function setUp() public {}
+
+    function run() public {
+        uint deployerPrivateKey = vm.envUint("PRIVATE_KEY");
```

Comment:
Do we have a `.env` file where this is set? Or does it just have a default value?

Reply:
add `.env.example`. it set `PRIVATE_KEY` to anvil's first account.